include/swift/AST/Expr.h

```diff
@@ -4553,7 +4553,7 @@ class ApplyExpr : public Expr {
     Bits.ApplyExpr.ShouldApplyDistributedThunk = flag;
   }
 
-  ValueDecl *getCalledValue() const;
+  ValueDecl *getCalledValue(bool skipFunctionConversions = false) const;
```

Comment:
Maybe a better name would be `lookThrough{Function}Conversions` just like we have for optionals? Also, I think the flag deserves a comment explaining the implications...